package.json

```diff
@@ -19,7 +18,7 @@
   }
 , "repository":
   { "type": "git"
-  , "url":  "http://github.com/agnat/node_mdns.git"
+  , "url":  "http://github.com/achingbrain/node_mdns.git"
```

Comment:
Please undo the changes to package.json. They do not belong into the PR. Note, that you don't need to republish a package just to get your PR in. Use the tarball or git URL of your PR in _your_ package.json. Much less noisy... See the last paragraph of the [Installation Section](https://github.com/agnat/node_mdns#installation) of the README for an example.